composer.json

```diff
@@ -37,14 +37,14 @@
         "ekino/phpstan-banned-code": "^0.3.1",
         "ergebnis/phpstan-rules": "^0.15.2",
         "infection/infection": "^0.17.3",
-        "phpstan/phpstan": "^0.12.45",
+        "phpstan/phpstan": "^0.12.49",
```

Comment:
The [master branch in your fork](https://github.com/proggga/paratest/tree/master) is way outdated: you should pull the latest commits from upstream, and then rebase the branch onto it, so your PRs don't get polluted by this kind of changes

Reply:
fixed